client/utils/log.c

```diff
@@ -321,19 +321,24 @@ extern int xdag_set_log_level(int level)
 
 #define REG_(name) sprintf(buf + strlen(buf), #name "=%llx, ",(unsigned long long)name##_sig(uc))
 
-#else
+#elif defined (__linux__)
```

Comment:
Any issue with windows?

Reply:
It is under  #if !defined(_WIN32) && !defined(_WIN64)  (line 290)
so it isn't (and wasn't) compiled in windows at all.